src/styles/blocks/price.scss

```diff
@@ -0,0 +1,53 @@
+.price {
+  &__wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    align-items: center;
+    margin-block: 24px 16px;
+    margin-inline: $margin-inline;
+  }
+
+  &__text {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $other-text-color;
+  }
+
+  &__money {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__link {
+    display: block;
+    margin-inline: $margin-inline;
+
+    background-color: $button-color;
+    height: 40px;
+    border-radius: 5px;
+
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    align-content: center;
+    color: $background-color-special-white;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    transition: all 300ms;
+
+    &:hover {
+      box-sizing: border-box;
+      color: $button-color;
+      background-color: $background-color-special-white;
+      border: 1px solid $button-color;
```

Comment:
The variables `$button-color` and `$background-color-special-white` are used but not defined. Please define these variables to ensure the hover styles are applied correctly.